test-cargo-miri/build.rs

```diff
@@ -11,6 +16,8 @@ fn not_in_miri() -> i32 {
 
 fn main() {
     not_in_miri();
+    // Make sure `--print=cfg` lists `miri`.
```

Comment:
What is that `--print=cfg` that your tests keep talking about?

Reply:
I mean `miri --print=cfg`:
```
$ miri --print=cfg
debug_assertions
miri
...
$ MIRI_BE_RUSTC=host miri --print=cfg
debug_assertions
panic="unwind"
...
```
https://github.com/rust-lang/miri/blob/7b2e325728e800664e55a7b91bac8b8b8378ab74/cargo-miri/bin.rs#L647

I think Cargo interpret that output to set `CARGO_CFG_*` environment variables: https://doc.rust-lang.org/nightly/cargo/reference/environment-variables.html#environment-variables-cargo-sets-for-build-scripts

Maybe I should write "`miri --print=cfg`"?

Reply:
I did not know about `CARGO_CFG_`. The `--print` part seems like an implementation detail, but a relevant one for getting this right in Miri. So maybe briefly explain that in the comment?
```
// Cargo calls `miri --print=cfg` to populate the `CARGO_CFG_*` env vars; make sure that the "miri" flag is set.
```
(or "is not set for proc macros")